docs/quick-start.md

```diff
@@ -330,6 +331,10 @@ Note that Spark artifacts are tagged with a Scala version.
   <name>Simple Project</name>
   <packaging>jar</packaging>
   <version>1.0</version>
+  <properties>
```

Comment:
I see two other instances in structured-streaming-programming-guide.md around line 1044.
If you can address those and remove this stanza, I'll merge